apps/gitness/src/pages/repo-sandbox/repo-sandbox-summary.tsx

```diff
@@ -176,93 +208,106 @@ export const RepoSandboxSummaryList: React.FC = () => {
       )
   }
   return (
-    <SandboxLayout.Main hasLeftPanel hasHeader hasSubHeader>
-      <SandboxLayout.Columns columnWidths="1fr 220px">
-        <SandboxLayout.Column>
-          <SandboxLayout.Content>
-            <ListActions.Root>
-              <ListActions.Left>
-                <ButtonGroup.Root>
-                  <BranchSelector
-                    size="default"
-                    name={selectedBranch}
-                    branchList={branchList}
-                    selectBranch={branch => selectBranch(branch)}
-                  />
+    <>
+      <SandboxLayout.Main hasLeftPanel hasHeader hasSubHeader>
```

Comment:
ideally most of this structure should be in playground components. but we can take that up separately.